epoch1/cuda/ee_mumu/SubProcesses/Makefile

```diff
@@ -1,33 +1,50 @@
-LIBDIR =../../lib
-INCDIRS=-I. -I../../src -I../../../../../tools/
-MODELLIB=model_sm
-CXXFLAGS= -O3 -std=c++11 $(INCDIRS) $(USE_NVTX) -Wall -Wshadow ${MGONGPU_CONFIG}
-LIBFLAGS= -L$(LIBDIR) -l$(MODELLIB)
-CXX=g++
-NVCC?=$(shell which nvcc)
-ifeq (, $(NVCC))
-	NVCC=@echo No nvcc to run nvcc
-else
-	USE_NVTX?=-DUSE_NVTX
+CXX  ?= g++
+NVCC ?= $(shell which nvcc 2>/dev/null)
+
+LIBDIR   = ../../lib
+TOOLSDIR = ../../../../../tools/
+INCFLAGS = -I. -I../../src -I$(TOOLSDIR)
+MODELLIB = model_sm
+OPTFLAGS = -O3
+CXXFLAGS = $(OPTFLAGS) -std=c++11 $(INCFLAGS) $(USE_NVTX) -Wall -Wshadow $(MGONGPU_CONFIG)
+LIBFLAGS = -L$(LIBDIR) -l$(MODELLIB)
+
+#Export CUDA_HOME to point to a cuda installation
+ifdef CUDA_HOME
+	NVCC       ?=   $(CUDA_HOME)/bin/nvcc
+	CUINC       = -I$(CUDA_HOME)/include/
+	CULIBFLAGS  = -L$(CUDA_HOME)/lib64/
+endif
+
+ifneq ($(NVCC),)
 	CUARCHNUM=70
 	#CUARCHNUM=61 #(For Pascal Architecture Cards)
```

Comment:
I leave this comment here about changes which actually came from PR #59.

My emacs gives me a warnings about indented comments.
As I got used to avoiding indeneted comments, I thought that make would fail, actually it does not.
But I just understood that there is a reason emacs complains, indented comments are parsed and executed: https://stackoverflow.com/a/18363477

Would you agree to avoiding indented comments?
Otherwise I can also tweak my emacs (though I'd rather not)

By the way, note that those definitions like CUARCHNUM=70 do NOT need to be indented. The easiest is to actually bring everything back to the first column. Indentation is really only needed for excuting build sequences.

Reply:
I didn't program `make` in a looong time. I also see vim screwing up the syntax colouring when you indent, so I might as well unindent all bodies of the conditionals. I will have a look tomorrow.

Reply:
Ok, problem understood:
In *recipes*, indented comments are indeed parsed as shell commands.

Here, we are not in a recipe, so indentation is possible but not required. What's "wrong", though, is that the blocks should be indented with spaces and not with tabs.

Reply:
We would have to agree on an indentation width, though, if we chose to indent the body of conditionals.

Reply:
@valassi Can you check if emacs also complains if you indent a comment with spaces? That should be "legal" in make.

Reply:
Hi @valassi , following the discussion below I presume this issue can be marked as resolved?